client/my-sites/pages/main.jsx

```diff
@@ -76,7 +77,7 @@ class PagesMain extends React.Component {
 	render() {
 		const { siteId, search, status, translate, queryType, author } = this.props;
 		const postStatus = mapPostStatus( status );
-
+		const allSites = siteId ? 1 : 2;
```

Comment:
I'm guessing you didn't mean to commit this line!

Reply:
🤔 I imagine this was intended to be the condition to be used to decide if the `subHeaderText` below should be singular or plural, although the variable name doesn't necessarily make that clear. It does, in fact, mirror the existing translation which uses `siteId` for that same condition.

Maybe refactoring this slightly would make it easier to read by removing this line and down below specifying something like:

```
//...
count: siteId ? 1 : 0
//...
```

In this way you're using `0` when the page has no single site selected, which should (I hope) still provide the plural version of the translation without having to resort to the otherwise arbitrary `2`.

Reply:
I think we settled with `allSites` for the naming b/c the plural subheader is for the "All Sites Mode". Maybe just a comment above this line explaining the variable would be enough.

Reply:
Ah, apologies - looking at this again, I see what's going on here now. I assumed it was for testing the pluralised translations, but I didn't notice the check for `siteId` was removed.